src/PrizeVault.sol

```diff
@@ -862,7 +888,12 @@ contract PrizeVault is TwabERC20, Claimable, IERC4626, ILiquidationSource, Ownab
 
         _mint(_receiver, _shares);
```

Comment:
If there is a total supply overflow, it seems that it would occur on this line, no?  The check should be before the mint so that the user gets a clean error